pwndbg/chain.py

```diff
@@ -75,17 +75,29 @@ def format(value, limit=LIMIT, code=True, offset=0, hard_stop=None, hard_end=0):
         A string representing pointers of each address and reference
         Strings format: 0x0804a10 —▸ 0x08061000 ◂— 0x41414141
     """
-    limit = int(limit)+1
+    limit = int(limit)
```

Comment:
The docs for `format` say:

> `limit(int): number of valid pointers`

However if limit is 0 it will increment it to 1 and try to dereference the address once (during `get`), which is unwanted.